set.mm

```diff
@@ -531068,66 +531362,81 @@ elements in a zero ring (see ~ 0ringirng ).  (Contributed by Thierry
-      MUFZUNUJZUWIUEUFZNUWIUFZUIUJZUWLUWMOPZUVARZGSUHZTZUOZNUWIUCULZUPZUQPZUMUF
-      ZUNUJZUXBUEUFZNUXBUFZUIUJZUXEUXFOPZQUDULZUGPZRZUDSUHZTZUOUVHUVIUOUCURDCUA
-      UBUVJVDRZUVLUWAUVSUWHUXNUVKUVTNUNUVJVDUMUSUTUXNUVOUWDUVRUWGUXNUVMUWBUVNUW
-      CUIUVJVDUEUSZNUVJVDVEZVAUXNUVQUWFGSUXNUVPUWEUVAUXNUVMUWBUVNUWCOUXOUXPVBVC
-      VFVGVHUAUBVIZUVLUWKUVSUWRUXQUVKUWJNUNUVJUWIUMUSUTUXQUVOUWNUVRUWQUXQUVMUWL
-      UVNUWMUIUVJUWIUEUSZNUVJUWIVEZVAUXQUVQUWPGSUXQUVPUWOUVAUXQUVMUWLUVNUWMOUXR
-      UXSVBVCVFVGVHUVJUXBRZUVLUXDUVSUXMUXTUVKUXCNUNUVJUXBUMUSUTUXTUVOUXGUVRUXLU
-      XTUVMUXEUVNUXFUIUVJUXBUEUSZNUVJUXBVEZVAUXTUVRUXHUVARZGSUHUXLUXTUVQUYCGSUX
-      TUVPUXHUVAUXTUVMUXEUVNUXFOUYAUYBVBVCVFUYCUXKGUDSGUDVIUVAUXJUXHUUTUXIQUGVJ
-      VKVLVMVGVHUVJDRZUVLUVHUVSUVIUYDUVKUVGNUNUVJDUMUSUTUYDUVOUVFUVRUVCUYDUVMUU
-      QUVNUURUIUVJDUEUSZNUVJDVEZVAUYDUVQUVBGSUYDUVPUUSUVAUYDUVMUUQUVNUUROUYEUYF
-      VBVCVFVGVHJAUWAUWHANNUNUJZUWAUYGVPANVNVQVOUVTNNUNVRVSVTWAAUWIURCWBZWCZTZU
-      WTURWCZTZUWIVDRZUWLUWTCUJZWDZTZUWSTZUXDUXMUYQUXDTZUXMUWIVDUYRUYMTZUXGUXLU
-      YSUWTUWTUXEUXFUIUYSUYKUWTUWTUIUJUYJUYKUYOUWSUXDUYMWEZUWTWFWHUYRUWIURWGZWC
-      ZUYMUYKUXEUWTRZUYRURUWICAUYIUYKUYOUWSUXDWEZWIZUYTUWTURUWIWJZWKZUYSUXFNUXA
-      UFZUWTUYSNUXBUXAUYSUXBVDUXAUQPUXAUYSUWIVDUXAUQUYRUYMWLWMUWTWNWOWPUYSUYKVU
-      HUWTRUYTUWTURWQWHWRZWSUYSUXKUXHWTRUDNSUXINRZUXJWTUXHVUJUXJQNUGPZWTUXINQUG
-      VJQXAWCVUKWTRXBQXCXDXEVKNSWCUYSXKVOUYSUXHUWTUWTOPZWTUYSUXEUWTUXFUWTOVUGVU
-      IVBUYSUYKVULWTRUYTUWTXFWHWRXGXHUYRUWIVDYBZTZUXGUXLVUNUWTUWMUXEUXFUIVUNUWT
-      UWLUIUJZUWNUWTUWMUIUJVUNUWOQUKULZUGPZRZVUOUKSVUNVUPSWCZTZVURTZVUOUWTUWLOP
-      ZQRZVVAUYNVUOVVCTZVVAUYMUYNUYLUYOUWSUXDVUMVUSVURXIVVAUWIVDUYRVUMVUSVURXJZ
-      XLXMVVAUWLURWCZUYKUYNVVDXNVVAVUBVUMVVFUYRVUBVUMVUSVURVUEXOZVVEURUWIXPXQUY
-      JUYKUYOUWSUXDVUMVUSVURXRZEULZFULZUIUJZVVIVVJOPZQRZTZVVDFEUWLUWTCURUREUCVI
-      ZVVJUWLRZVVNVVDXNVVOVVPTZVVKVUOVVMVVCVVIUWTVVJUWLUIXSVVQVVLVVBQVVIUWTVVJU
-      WLOXTVCVGYAIYCXQYDZYEZVUNUWQVURUKSUHVUNUWNUWQVUNUWKUWRUYRUYIVUMUWKVUDUWIU
-      YHYFYGZUYPUWSUXDVUMXJYMZYHUWPVURGUKSGUKVIUVAVUQUWOUUTVUPQUGVJVKVLYIZYJVUN
-      UWNUWQVWAYEZUWTUWLUWMYKXQVUNVURVUCUKSVVAVUBUYKVUCVVGVVHVUFXQZVWBYJVUNVURU
-      XFUWMRZUKSVVAVUBUXAVUAWCUWKVWEVVGVVAUWTURVVHYLVUNUWKVUSVURVVTYNUWIUXAURYO
-      YPZVWBYJWSVUNVURUXLUKSVVAUXKUXHQVUPWTYQPZUGPZRUDVWGSUXIVWGRUXJVWHUXHUXIVW
-      GQUGVJVKVVAVUPWTVUNVUSVURUUAZWTSWCVVAUUDVOUUBVVAUXHUWTUWMOPZVVBUWOYRPZVWH
-      VVAUXEUWTUXFUWMOVWDVWFVBVVAVUOUWNVWJVWKRVVSVUNUWNVUSVURVWCYNUWTUWLUWMUUCX
-      QVVAQVUQYSPZVUQQYSPVWKVWHVVAQVUQVVAUUEZVVAQVUPVWMVWIUUFUUGVVAVWKQVUQYRPZV
-      WLVVAVVBQUWOVUQYRVVAVUOVVCVVRYHVUTVURWLVBVVAQYTWCZVUQYTWCVWNVWLRVWOVVAUUN
-      VOZVVAQVUPVWPVWIUUHQVUQUUIXQWRVVAQVUPVWMVWIUUJUUKUULXGVWBYJXHUUMUUOUUPYMY
-      HAUVBUVEGSAUUSUVDUVAAUUQHUURBOLKVBVCVFYD $.
+    $( In a non-empty chain ` T ` of quadratic field extensions, the degree of
+       the final extension is always some power of two.  (Contributed by
```

Comment:
I think "some power of two" can be "a power of two". Also earlier you wrote "power of ` 2 `", so maybe it's better to be consistent ("two" vs `2`).